app.js

```diff
@@ -22,90 +17,148 @@ function loopStores(){
 
 //LOOP THE HOURS OF DAY * loopAvgCookies();
 function aDay(x){
-  // for(var y = 0; y < stores.length; y++){
-  //   return stores[y];
-  // };
   return x.avgHourly();
 };
 
-// function avgHourly(){
-//   return Math.floor(Math.random() * ((this.maxCust - this.minCust + 1) + this.minCust) * this.avgSale);
-// };
-
+//Cookie store repo
 function CookieStore(minCust, maxCust, avgSale) {
   this.minCust = minCust;
   this.maxCust = maxCust;
-  this.avgSale = maxSale;
+  this.avgSale = avgSale;
 }
+//Our Stores
+var onePike = new CookieStore(23, 65, 6.3);
+var seaTac = new CookieStore(3, 24, 1.2);
+var seaCenter = new CookieStore(11, 38, 3.7);
+var capHill = new CookieStore(20, 38, 2.3);
+var alki = new CookieStore(2, 16, 4.6);
 
 CookieStore.prototype.avgHourly = function(){
   return Math.floor(Math.random() * ((this.maxCust - this.minCust + 1) + this.minCust) * this.avgSale);
 };
 
-//--STORE OBJECTS BELOW--
-// var onePike = new CookieStore(23, 65, 6.3,);
-var onePike = {
-  minCust: 23,
-  maxCust: 65,
-  avgSale: 6.3,
-  avgHourly: function(){
-    return Math.floor(Math.random() * ((this.maxCust - this.minCust + 1) + this.minCust) * this.avgSale);
-  },
-};
+var mainTable = document.getElementById('main-table');
+var cookieListEl = document.getElementById('cookie-list');
+var bodyAppendEl = document.getElementById('main-table');
 
-var seaTac = {
-  minCust: 3,
-  maxCust: 24,
-  avgSale: 1.2,
-  avgHourly: function(){
-    return Math.floor(Math.random() * ((this.maxCust - this.minCust + 1) + this.minCust) * this.avgSale);
-  }
-};
+var tableHead = document.createElement('thead');
+mainTable.appendChild(tableHead);
 
-var seaCenter = {
-  minCust: 11,
-  maxCust: 38,
-  avgSale: 3.7,
-  avgHourly: function(){
-    return Math.floor(Math.random() * ((this.maxCust - this.minCust + 1) + this.minCust) * this.avgSale);
-  }
-};
+// spacer block for thead
+var spacerTh = document.createElement('th');
+tableHead.appendChild(spacerTh);
 
-var capHill = {
-  minCust: 20,
-  maxCust: 38,
-  avgSale: 2.3,
-  avgHourly: function(){
-    return Math.floor(Math.random() * ((this.maxCust - this.minCust + 1) + this.minCust) * this.avgSale);
-  }
-};
-
-var alki = {
-  minCust: 2,
-  maxCust: 16,
-  avgSale: 4.6,
-  avgHourly: function(){
-    return Math.floor(Math.random() * ((this.maxCust - this.minCust + 1) + this.minCust) * this.avgSale);
-  }
-};
+for(var h = 0; h < timeDay.length; h++){
+  var eHeaderItem = document.createElement('th');
+  eHeaderItem.textContent = timeDay[h];
+  tableHead.appendChild(eHeaderItem);
+}
 
+//
 for(var v = 0; v < stores.length; v++){
-  var loopDl = document.createElement('dl');
-  hOneStart.appendChild(loopDl);
-  var loopedId = document.getElementById(stores[v]);
-  var hOne = document.createElement('dt');
-  // hOne.setAttribute('class', 'slideOut');
-  hOne.setAttribute('id', storesHtml[v]);
-  hOne.textContent = stores[v];
-  loopDl.appendChild(hOne);
+  var loopTr = document.createElement('tr');
```

Comment:
from the assignment instructions: Each cookie stand location should have a separate render() method that creates and appends its row to the table